sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/DateTimeFormatterHelper.scala

```diff
@@ -99,12 +99,35 @@ private object DateTimeFormatterHelper {
     new DateTimeFormatterBuilder().parseCaseInsensitive()
   }
 
-  def toFormatter(builder: DateTimeFormatterBuilder, locale: Locale): DateTimeFormatter = {
-    builder
-      .parseDefaulting(ChronoField.MONTH_OF_YEAR, 1)
-      .parseDefaulting(ChronoField.DAY_OF_MONTH, 1)
-      .parseDefaulting(ChronoField.MINUTE_OF_HOUR, 0)
-      .parseDefaulting(ChronoField.SECOND_OF_MINUTE, 0)
+  def setDefaulting(builder: DateTimeFormatterBuilder,
+                    locale: Locale,
+                     isWeekBased: Boolean = false): DateTimeFormatterBuilder = {
```

Comment:
Please, change the indentation as in the style guide https://github.com/databricks/scala-style-guide#spacing-and-indentation